packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -335,6 +361,14 @@ class ChartController extends DisposableController
     recomputeLabels(refresh: true);
   }
 
+  /// Clear all data in the chart.
+  void reset() {
+    for (var index = 0; index < traces?.length;  index++) {
```

Comment:
nit: this can be a `for (var trace in traces)`

Reply:
Done.